libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -219,7 +219,8 @@ void NavEKF2_core::ResetHeight(void)
 
     // Reset the vertical velocity state using GPS vertical velocity if we are airborne
     // Check that GPS vertical velocity data is available and can be used
-    if (inFlight && !gpsNotAvailable && frontend->_fusionModeGPS == 0 && !frontend->inhibitGpsVertVelUse) {
+    if (inFlight && !gpsNotAvailable && frontend->_fusionModeGPS == 0 &&
+        dal.gps().have_vertical_velocity()) {
```

Comment:
So I guess this means a small change in behaviour because previously a ResetHeight would have resulted in the vehicle's vertical velocity being reset to zero if the GPS didn't provide velocities but not the vertical velocity will not be reset at all?

I guess that even if we don't reset the velocity we still want to reset the variances?  Maybe this is a question for @priseborough

Reply:
There is no change of behaviour that I can see. Previously frontend->_fusionModeGPS would not have been zero, so fell through to the onGround check. It does the same now.

Reply:
Ah right, ok then. seems good to me.